Project.toml

```diff
@@ -1,11 +1,12 @@
 name = "ForwardDiff"
 uuid = "f6369f11-7733-5829-9624-2563aa707210"
-version = "1.0.1"
+version = "1.0.2"
 
 [deps]
 CommonSubexpressions = "bbf7d656-a473-5ed7-a52c-81e309532950"
 DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
 DiffRules = "b552c78f-8df3-52c6-915a-8e097449b14b"
+GPUArraysCore = "46192b85-c4d5-4398-a991-12ede77f4527"
```

Comment:
IMO we should update the Julia compat to 1.10 instead of enforcing a new dependency on users of Julia < 1.10.
```suggestion
```